test/unit/step-collection-test.js

```diff
@@ -20,20 +21,20 @@ describe('Steps', function() {
     this.run = (done) => {
       runCount += 1;
       done();
-    }
-  }
+    };
+  };
 
   beforeEach(function() {
     this.sinon = sinon.sandbox.create();
     runCount      = 0;
-    Steps.Step    = Step;
+    StepCollection.Step = Step;
     mockStream    = new OutputStream();
     outputStream  = mockStream.stream;
     logger        = new Logger(outputStream);
     compiledSteps = compiledFeature.document.feature.children[0].steps;
     world         = new World({
       logger: logger,
-      stepDefinitions: stepDefinitions
+      stepDefinitions: StepDefinitionCollection
```

Comment:
Closing this PR. However, there was an issue here.

Here is the original line:
https://github.com/baccigalupi/cucaroo/blob/master/test/unit/steps-test.js#L36

You can see that `stepDefinitions` isn't declared or required anywhere. Not sure how V8 wasn't balking at that. 

At any rate, even those `world` will apply `stepDefinitions` passed to the constructor, it will immediately overwrite it: https://github.com/baccigalupi/cucaroo/blob/master/test/unit/steps-test.js#L36

Maybe I'm missing something?